x-pack/plugins/apm/public/components/app/service_groups/service_groups_list/service_group_card.tsx

```diff
@@ -58,15 +83,15 @@ export function ServiceGroupsCard({
         {!hideServiceCount && (
           <EuiFlexItem>
             <EuiText size="s">
-              {servicesCount === undefined ? (
+              {serviceGroupCounts === undefined ? (
                 <>&nbsp;</>
```

Comment:
Is it really needed? Can we just render the translation if `serviceCount`is defined?

Reply:
This was in place for 8.5 in order to avoid a jump in height after service counts were loaded. But we may want to re-think the loading state for these cards of service groups.